tests/e2e/test_spyre_static_batching_limits.py

```diff
@@ -0,0 +1,68 @@
+"""Verification of handling prompt length exceeding warmup shapes
+
+Run `python -m pytest tests/test_spyre_max_prompt_length.py`.
+"""
+
+import pytest
+from spyre_util import (create_text_prompt, get_spyre_backend_list,
+                        get_spyre_model_list, patch_warmup_shapes)
+from vllm import LLM, SamplingParams
+
+
+@pytest.mark.parametrize("model", get_spyre_model_list())
+@pytest.mark.parametrize(
+    "warmup_shapes",
+    [[(64, 20, 4)], [(64, 20, 4),
+                     (128, 20, 4)]])  # (prompt_length/new_tokens/batch_size)
+@pytest.mark.parametrize("backend", get_spyre_backend_list())
+@pytest.mark.parametrize("vllm_version", ["V0", "V1"])
```

Comment:
should we be using `run_with_both_engines` fixture instead? Not sure why it's there but not used anywhere